index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you to add a title :smile: to make your project look better.

Reply:
- Good use of optional, But I would suggest you reduce the use of the laughing emoji as the student you are reviewing might read the message wrongly and thinks you are making fun of them, If you would use an emoji, I will suggest a thumbs up :+1:  or a  :100: